apps/music/js/ui/views/player_view.js

```diff
@@ -1039,6 +1053,21 @@ var PlayerView = {
           this.next();
         }
         break;
+      case 'keypress':
+        // The standard accessible control for sliders is arrow up/down keys.
+        // Our screenreader synthesizes those events on swipe up/down gestures.
+        // Currently, we only allow screen reader users to adjust sliders with a
+        // constant step size (there is no page up/down equivalent). In the case
+        // of videos, we make sure that the maximum amount of steps for the
+        // entire duration is 20, or 2 second increments if the duration is less
+        // then 40 seconds.
+        var step = Math.max(this.audio.duration / 20, 2);
+        if (evt.keyCode == evt.DOM_VK_DOWN) {
+          this.seekAudio(this.audio.currentTime - step);
+        } else if (evt.keyCode == evt.DOM_VK_UP) {
+          this.seekAudio(this.audio.currentTime + step);
+        }
+        break;
```

Comment:
@eeejay This looks exactly the same as those logic in video so should be fine, just curious why they are mapping to up/down instead of left/right keys, cause usually the fastforward/rewind icons goes left and right.